asf_search/search/baseline_search.py

```diff
@@ -84,18 +84,22 @@ def get_stack_opts(
         raise ASFBaselineError(f'Requested reference product needs a baseline stack ID but does not have one: {reference.properties["fileID"]}')
 
     # build a stack from scratch if it's a non-precalc dataset with state vectors
-    if reference.properties['platform'] in [PLATFORM.SENTINEL1A, PLATFORM.SENTINEL1B]:
-        stack_opts.platform = [PLATFORM.SENTINEL1A, PLATFORM.SENTINEL1B]
+    if reference.properties['platform'].upper() in [PLATFORM.SENTINEL1A.upper(), PLATFORM.SENTINEL1B.upper()]:
+        stack_opts.platform = [reference.properties['platform'], PLATFORM.SENTINEL1A, PLATFORM.SENTINEL1B]
```

Comment:
Isn't this just adding one of those two names that are already in the list by way of constants? I believe CMR will disregard capitalization differences, so any issues caused by that is purely at our end.